From bfba4b0e0126aa2229ce6ea16f333557e31492a9 Mon Sep 17 00:00:00 2001
From: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
Date: Sun, 6 Apr 2025 20:22:57 +0200
Subject: [PATCH] criu/include/mman: define MADV_GUARD_INSTALL

Signed-off-by: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
(cherry picked from commit fce491113bcb5bfe95e078ba92e2601b7f671c23)

Conflicts:
	criu/include/mman.h
---
 criu/include/mman.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/criu/include/mman.h b/criu/include/mman.h
index 8ca71fadf..5138cbe6c 100644
--- a/criu/include/mman.h
+++ b/criu/include/mman.h
@@ -13,5 +13,8 @@
 #ifndef MADV_DONTDUMP
 #define MADV_DONTDUMP 16
 #endif
+#ifndef MADV_GUARD_INSTALL
+#define MADV_GUARD_INSTALL 102
+#endif
 
 #endif /* __CR_MMAN_H__ */
-- 
2.34.1

